src/assets-webkit/styles/serlo-tailwind.css

```diff
@@ -115,6 +115,17 @@
   .serlo-img {
     @apply inline h-auto max-w-full rounded-md;
   }
+  .serlo-img-error-placeholder {
+    @apply relative min-h-[100px] min-w-[100%];
+  }
+  .serlo-img-error-placeholder::before {
+    @apply absolute left-0 top-[50%] h-full w-full translate-y-[-50%] border-2 border-editor-primary-100 bg-slate-100;
+    content: '';
+  }
+  .serlo-img-error-placeholder::after {
+    @apply absolute left-0 top-1 top-1 block h-full w-full;
+    content: '\2639'' ' attr(alt);
+  }
```

Comment:
the styles here are mainly for components that are used in multiple places and need overriding attributes. For one time usage you can directly use tailwind.

Reply:
maybe relevant: https://tailwindcss.com/docs/content  :)